lib/product-import.coffee

```diff
@@ -230,6 +232,52 @@ class ProductImport
       else
         false
 
+  _updateProductRepeater: (prodToProcess, existingProduct) ->
+    repeater = new Repeater {attempts: 5}
```

Comment:
And regarding the retries count - I just wanted to use some nice number .. What do you think? How many retries should it have?